src/internal/snapshot/recovery.go

```diff
@@ -214,6 +215,65 @@ func (s *Snapshotter) dumpTx(ctx context.Context, tx *pachsql.Tx, opts CreateSna
 	return id, fh, nil
 }
 
+func (s *Snapshotter) pinDatabaseDump(ctx context.Context, fh io.Reader, id SnapshotID) (retErr error) {
+	ctx, done := log.SpanContext(ctx, "pinDatabaseDump")
+	defer done(log.Errorp(&retErr))
+
+	var closedFileSet, pinnedFileSet bool
```

Comment:
nit, fileSet -> fileset. This is a bit inconsistent in the codebase, but we are gradually moving towards fileset.

Reply:
Removed these two variables based on advice from future comments, but that's good to know.  I like `fileset` better than `fileSet` too.